cmd/coder/urls.go

```diff
@@ -35,47 +36,52 @@ var urlAccessLevel = map[string]string{
 
 func portIsValid(port string) bool {
 	p, err := strconv.ParseUint(port, 10, 16)
-	if p < 1 {
+	if p < 1 || p > 65535 {
 		// port 0 means 'any free port', which we don't support
 		err = strconv.ErrRange
 	}
 	if err != nil {
-		fmt.Println("Invalid port")
+		flog.Error("Invalid port")
 	}
 	return err == nil
```

Comment:
In cdr/enterprise the convention appears to prefer multiple return points where errors are first detected; if we want to follow that for the coder-cli repo I can update everything I touched in this PR to use that style.